web/src/app/admin/AdminRouter.js

```diff
@@ -1,14 +1,15 @@
 import React from 'react'
-import { Switch, Route } from 'react-router-dom'
+import { Switch, Route, Redirect } from 'react-router-dom'
 import { GenericError, PageNotFound } from '../error-pages/Errors'
 import AdminConfig from './AdminConfig'
 import RequireConfig from '../util/RequireConfig'
 
 export const AdminRouter = () => (
   <Switch>
+    <Route exact path='/admin' render={() => <Redirect to='/admin/config' />} />
```

Comment:
Can we move this to `routes.js`?

We already handle most redirects inside `renderRoutes`, and could add something like this:
```js
if (cfg.subRoutes && cfg.subRoutes.length) {
    routes.push(
      <Redirect
        key={`redir_sub_${idx}`}
        strict
        exact
        from={path.replace(/\/$/, '')}
        from={cfg.subRoutes[0].path}
      />,
    )
}
```

That way all subroute-items will behave the same and all redirect logic is in a single place.